packages/babel-parser/test/fixtures/v8intrinsic/_errors/not-in-call-expression/input.js

```diff
@@ -0,0 +1,2 @@
+const i = %DebugpPrint;
```

Comment:
typo: `%DebugPrint`

(not that it really matters for the purposes of the test case)

Reply:
Ahh, good catch. Fixed and pushed.